package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node",
+    "dev": "ts-node src/app.ts" 
```

Comment:
Here we can get fancier. `ts-node-dev --poll --no-notify src/app.ts` to have hot-reload when in dev.
I prefer the `watch` keyword instead of `dev`. But that is up to you.

_We could use `nodemon` but we discovered that ts-node-dev is way smarter and faster_